api/typescript/src/core/lib/enclaves/enclave_context.ts

```diff
@@ -304,6 +304,8 @@ export class EnclaveContext {
         return ok(serviceContext);
     }
 
+    // TODO: Add getServiceContexts
```

Comment:
Todo for the future?

Reply:
Yes i want to push the current changes asap so they can be used during user interviews.